dead-simple-k8s/hostIP.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+
+sleep 10
+curl -s "${K8S_API:-localhost:8001}/api/v1/namespaces/$NS/pods/$POD_NAME" | jq '.status.hostIP' | sed 's/"//g'
```

Comment:
Wait.  I think we can just use the [downward API](http://kubernetes.io/docs/user-guide/downward-api/) for this!

Something like:

```yaml
containers:
- ...
  env:
  - name: HOST_IP
          valueFrom:
            fieldRef:
              fieldPath: status.hostIP
```